woocommerce/woocommerce-android
# Order Detail: Pull to Refresh

Fixes #182. This PR implements Pull to Refresh support in the Order Details UI.

### Changes:
- Added skeleton view for Order Detail
- Added logic to refresh order detail page when swipe to refresh is called.
- Added new event `*_order_detail_pulled_to_refresh`.
- Added unit tests for `OrderDetailPresenter` class.

### Testing:
1. Open the app and click on the `Orders` tab.
2. Click on any order from the list
3. Open WooCommerce in your browser. Edit something in the order you're visualising in Mobile
4. Perform the Pull to Refresh gesture
5. Verify the UI reflects the changes you've just performed.
6. Run `OrderDetailPresenterTest`

Update release notes:
